pdns/dnsdistdist/dnsdist-rules.hh

```diff
@@ -542,6 +543,22 @@ private:
   DNSName d_qname;
 };
 
+class QNameSetRule : public DNSRule {
+public:
+    QNameSetRule(const DNSNameSet names) : qname_idx(names) {}
```

Comment:
It doesn't really matter but I'm pretty sure we will get a warning from `cppcheck` otherwise:
```suggestion
    QNameSetRule(const DNSNameSet& names) : qname_idx(names) {}
```

Reply:
Fixed.